packages/lib/auth.ts

```diff
@@ -9,3 +9,19 @@ export async function verifyPassword(password: string, hashedPassword: string) {
   const isValid = await compare(password, hashedPassword);
   return isValid;
 }
+
+export function isPasswordValid(password: string) {
```

Comment:
Let's put this in `packages/lib` instead

Reply:
@zomars you mean in the ´index.ts´ file instead of ´auth.ts´?